types/react-relay/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-relay 1.3
+// Type definitions for react-relay 4.0
```

Comment:
can we move to react-relay 5.0.0?

Reply:
That is the plan, but I need to start using relay 5 and testing it. Might be able to get that all done today. Worst case, Monday.

Reply:
the only breaking change is internal api, won't change anything about types of public api